src/main/java/ru/curriculum/domain/directories/staffTable/StaffTable.java

```diff
@@ -0,0 +1,25 @@
+package ru.curriculum.domain.directories.staffTable;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "staff_table")
+@Getter
+@Accessors(fluent = true)
+@NoArgsConstructor
+@EqualsAndHashCode
+public class StaffTable {
```

Comment:
Why is StaffTable?  this is Staff or may be StaffPosition

Reply:
well i'll correct